### PR TITLE
ResNet, DenseNet and Basic Inception Model Completed

### PR DESCRIPTION
**Pull request by AnirudhAchal**
@geekswaroop Please review my PR whenever it suits you. Thanks!

### CODE REVIEW

File: Learning Phase/Anirudh/Neural_Networks/Inception_Neural_Network.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+        X_sidebranch1 = self.sideblock1(X_sidebranch1)
+
+        X = self.stage3_IBA_1(X)
+
+        X = self.stage4_ag(X)
+        X = X.view(training_examples, -1)
+        X = self.stage4_fc(X)
+        X = torch.log_softmax(X, dim = 1)
+
+        return X, X_sidebranch1
```
Comment (geekswaroop) [inline Learning Phase/Anirudh/Neural_Networks/Inception_Neural_Network.py]: Changes required here

**Review by geekswaroop — COMMENTED** [review]
Change inceptionv3 to support sum of 3 losses.

File: Learning Phase/Anirudh/Neural_Networks/Inception_Neural_Network.py (hunk 1 above)
Comment (AnirudhAchal) [reply Learning Phase/Anirudh/Neural_Networks/Inception_Neural_Network.py]: I have completed InceptionV3 with 3 losses and 6 stages. Please review it whenever it suits you.

**Comment by AnirudhAchal** [comment]
@geekswaroop Let me know if it's alright now

**Review by geekswaroop — COMMENTED** [review]
LGTM

**Pull request merged**